samtranslator/model/cloudwatch.py

```diff
@@ -20,3 +21,33 @@ class SyntheticsCanary(Resource):
         "Tags": PropertyType(False, list_of(is_type(dict))),
         "VPCConfig": PropertyType(False, is_type(dict)),
     }
+
+
+class CloudWatchAlarm(Resource):
```

Comment:
There are few more inconsistencies between this and actual Cloudwatch alarm CFN, can you please make sure that we are consistent here.